src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { Avatar, Grid, TextField, InputLabel } from '@material-ui/core';
+import { useStyles } from './styles/PersonalInfo.styles';
+import { IExpert } from '../../../old/lib/types';
+import { getCurrentUser } from '../../../old/lib/utilities/API/api';
+import { ERROR_404 } from '../../../old/lib/constants/routes';
+
+export const PersonalInfo: React.FC = () => {
+  const classes = useStyles();
+  const { expertId } = useParams<{ expertId: string }>();
+  const [loadedExpert, setLoadedExpert] = useState<IExpert>();
+  const history = useHistory();
+
+  const fetchExpert = useCallback(async () => {
+    try {
+      const expertResponse = await getCurrentUser();
+      setLoadedExpert(expertResponse.data);
+    } catch (error) {
+      history.push(ERROR_404);
+    }
+  }, [expertId]);
+
+  useEffect(() => {
+    fetchExpert();
+  }, [fetchExpert]);
+
+  const inputChangeHandler = (
+    event: { target: { value: string } },
+    inputIdetifier: string,
+  ) => {
+    const updatedExpert: IExpert | undefined = { ...loadedExpert } as IExpert;
+    updatedExpert[inputIdetifier] = event.target.value;
+    setLoadedExpert(updatedExpert);
+  };
+
+  return (
+    <form>
+      <Grid container spacing={6} className={classes.PersonalInfo}>
+        <Grid xs={4}>
+          <Avatar
+            alt="Pic"
+            className={classes.Avatar}
+            src={loadedExpert?.avatar}
+          />
+        </Grid>
+        <Grid item xs={4}>
+          <TextField
+            variant="outlined"
+            label="Прізвище"
```

Comment:
may we read/get labels and placeholder strings from i18n file?

Reply:
I assume we surely can. At the same time, I would have a separate ticket to improve this part and to do not hold up this epic branch. Please let me know if it still should be done in scope of current PR :)

Reply:
sure, up to you. My comments are rather suggestions.